rewrite-java/src/main/java/org/openrewrite/java/style/ImportLayoutStyle.java

```diff
@@ -443,6 +445,45 @@ public void claimImport(Block block, JRightPadded<J.Import> impoort) {
         }
     }
 
+    private static class ImportLayoutInfo {

```

Comment:
Not sure what the cleanest way to access the isPackageFoldable method (naming may be scrutinized).
The intent of the class is to allow AddImport (will be added separately) to prevent folding imports too.